registry.k8s.io/manifests/k8s-staging-coredns/promoter-manifest.yaml

```diff
@@ -1,6 +1,6 @@
 # google group for gcr.io/k8s-staging-coredns is k8s-infra-staging-coredns@kubernetes.io
 registries:
-- name: gcr.io/k8s-staging-coredns
+- name: coredns/coredns
```

Comment:
@pacoxu @dims what's the build process for Coredns ? 
Do we just mirror from `coredns/coredns`?